src/main/resources/com/google/api/codegen/LICENSE.snip

```diff
@@ -187,7 +187,7 @@
           same "printed page" as the copyright notice for easier
           identification within third-party archives.
     
-       Copyright [yyyy] [name of copyright owner]
+       Copyright 2020 Google LLC
```

Comment:
Thanks for contributing!

However I think the brackets are intentional as a few lines above, it says
```
To apply the Apache License to your work, attach the following
          boilerplate notice, with the fields enclosed by brackets "[]"
          replaced with your own identifying information. (Don't include
```

We've already applied such rule to all generated source code.

Reply:
It doesn't make it through rules_gapic. I just tested py_gapic_assembly_pkg.

Reply:
So we actually don't need it at all - as long as the generated source code have licenses as comments on the top, we don't need a separate license template.

Reply:
I'll revert the brackets, would be nice to keep the text aligned in the license template.

Reply:
Sounds good. Thanks!

Reply:
new commit with those changes reverted